ecs-cli/modules/cli/regcreds/create_task_execution_role.go

```diff
@@ -19,7 +19,7 @@ import (
 
 	iamClient "github.com/aws/amazon-ecs-cli/ecs-cli/modules/clients/aws/iam"
 	kmsClient "github.com/aws/amazon-ecs-cli/ecs-cli/modules/clients/aws/kms"
-	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/utils"
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/utils/regcreds"
```

Comment:
This was a problem with `readers` as well. `regcredsio` was an attempt to be more accurate (than "readers") re: pkg purpose,  but now that the pkg name is even closer to `/regcreds` I agree it's probably best to just go with `regcreds` to avoid said path confusion :P

Reply:
OR I can just rename the path to `regcredio` :D